style.css

```diff
@@ -71,6 +73,173 @@ h1 {
   color: #6070ff;
 }
 
+.all-project-section {
+  margin: 114px 24px 0;
+  display: flex;
+  flex-direction: column;
+  gap: 114px;
+}
```

Comment:
- [Optional] As demanded by the project requirements, kindly consider using a CSS grid to lay out the contents of the project section.

Reply:
Done